provisioning_virtual_machines_and_hosts/_topics/catalogs_and_services.md

```diff
@@ -372,6 +372,125 @@ Service methods have been split based on purpose.
 
 7.  Click **Save** or **Add** if you are adding this to a new instance.
 
+### Provisioning Dialog
+
+The following example shows how to create a Provisioning Dialog that modifies the parameters for provisioning a small, medium, or large service.
+
+The `provision-vm-small-med-large.asl` workflow runs for every provisioned VM and is responsible for modifying the VM size. This workflow is the `<ManageIQ>` service Provisioning Entry Point.
+
+You need to set the ManageIQ Provisioning service entry point to workflow script. For example, this workflow is run for every provisioned Virtual Machine (VM), and modifies the VM size.
+
+Use the following sections and follow them in order to provision a VM service, create Provisioning Dialog and modify the paramaters.
+
+#### Create a workflow (optional)
+
+**Note**: If you want to use the existing `provision-vm-small-med-large.asl` workflow script without any changes, you can skip this section and proceed to `Add a Workflow Repository` section.
+
+1. Fork `https://github.com/ManageIQ/workflows-examples` repository or create your own repository.
```

Comment:
@kbrock I thought we discussed putting this as a real example in the manageiq-content repo?  The reason for this was specifically to avoid all this external git repository stuff.

Reply:
Yes. I'll move across.
But we still need to explain to people how to point to an external repo and do this on their own.

What example would you like me to point to for that?

Reply:
Yeah that's a fair point.  We can probably leave as is then.